app/javascript/packages/spinner-button/spinner-button-element.ts

```diff
@@ -73,16 +78,19 @@ export class SpinnerButtonElement extends HTMLElement {
 
     window.clearTimeout(this.#longWaitTimeout);
     if (isVisible && Number.isFinite(this.longWaitDurationMs)) {
-      this.#longWaitTimeout = window.setTimeout(
-        () => this.handleLongWait(),
-        this.longWaitDurationMs,
-      );
+      this.#longWaitTimeout = window.setTimeout(this.#handleLongWait, this.longWaitDurationMs);
     }
   }
 
-  handleLongWait() {
+  #handleLongWait = () => {
```

Comment:
Taking more advantage of private class members, for clearer separate of public/private interface, and better bundle optimization ([related Slack thread](https://gsa-tts.slack.com/archives/C032KSPPQ/p1697035661315639)).